subprojects/file-watching/src/integTest/groovy/org/gradle/internal/watch/EnableFileSystemWatchingIntegrationTest.groovy

```diff
@@ -76,26 +76,6 @@ class EnableFileSystemWatchingIntegrationTest extends AbstractFileSystemWatching
         "--no-watch-fs"   | DISABLED_MESSAGE
     }
 
-    @Unroll
-    @SuppressWarnings('GrDeprecatedAPIUsage')
-    def "deprecation message is shown when using the old property to enable watching the file system (enabled: #enabled)"() {
```

Comment:
We should have a test to show that FS watching is enabled by default.

Reply:
Good point, added.